### PR TITLE
update publish workflow (close #588)

### LINKED ISSUE #588: Publish validation errors
untitled

### PR DESCRIPTION
**Pull request by miherlosev**
For publish we will use command `npm run publish-please` instead of `gulp publish`.
@inikulin @churkin

### CODE REVIEW

**Comment by churkin** [comment]
lgtm

**Comment by inikulin** [comment]
I'll fix publish-please guard issue, so it will work on node 4

**Comment by inikulin** [comment]
https://github.com/inikulin/publish-please/issues/31

**Comment by inikulin** [comment]
Published v2.1.0 - should fix the issue

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
FPR

File: package.json [hunk 1]
```diff
@@ -71,6 +75,10 @@
...
   ],
   "scripts": {
-    "test": "gulp travis"
-  }
+    "test": "gulp travis",
+    "publish-please": "publish-please",
+    "prepublish": "publish-please guard"
+  },
+  "readme": "ERROR: No README data found!",
+  "_id": "testcafe-hammerhead@8.3.1"
```
Comment (inikulin) [inline package.json]: Let's remove this `_id" field

**Comment by inikulin** [comment]
lgtm

**Pull request merged**